client/src/main/java/io/pravega/client/stream/impl/ControllerImplConfig.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import java.io.Serializable;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ControllerImplConfig implements Serializable {
```

Comment:
Yes, controller instances are created internally by the pravega admin and reader/writer clients.
Have included this config in case we want to override the defaults, which currently happens only in the test clients. I am fine with using internal defaults and exposing this config only to test clients. Let me know